test/single_layer/test_binaryoplayer.cpp

```diff
@@ -0,0 +1,219 @@
+﻿#include <vector>
+
+#include "gtest/gtest.h"
+#include "layers/BinaryOpLayer.hpp"
+#include "layers/Tensor.hpp"
+
+using namespace it_lab_ai;
+
+class BinaryOpLayerMulTests : public ::testing::Test {
```

Comment:
These are not `BinaryOpLayerMulTests` but rather `BinaryOpLayerTests` since different ops are being tested as a part of this class